common/scala/src/whisk/common/Logging.scala

```diff
@@ -42,25 +44,25 @@ trait Logging {
     def setComponentName(comp: String) =
         this.componentName = comp
 
-    def debug(from: AnyRef, message: String, marker: String = null)(implicit id: TransactionId = TransactionId.unknown) = {
+    def debug(from: AnyRef, message: String, marker: LogMarkerToken = null)(implicit id: TransactionId = TransactionId.unknown) = {
         val mark = id.mark(marker)
```

Comment:
Is it possible to add the markers you intend to use to the predefined list so you don't need to change all these to String (and continue to use LogMarkerToken instead)?